lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -436,7 +436,34 @@ span.ltx_framed       { display:inline-block; text-indent:0; } /* avoid padding/
 .ltx_nounicode    { color:red; }
 .ltx_nodisplay    { display:none; }
 /*======================================================================
- SVG (pgf/tikz ?) basics */
+ SVG (pgf/tikz & xy) basics
+ Layout depends heavily on TeX's fonts which are more compact than typical browser's fonts
+ */
+/* Use flex to center the FO's content (horizontally & vertically) over the FO's size */
+
+/* Use width 110% in a couple of places to inhibit unexpected line breaks
+  due to TeX/browser font differences */
+.ltx_foreignobject_container {
+    display:flex;
+    align-items:center;
+    justify-content:center;
+    height:100%;
+    width:110%;
+    position:relative; left:-5%;
+}
+/* line height 0 just for this layer, to keep the line-box compact */
+.ltx_foreignobject_content {
+    display:block;
+    line-height:0;
+}
+/*    line-height:revert;  Why doesn't this work? */
+.ltx_foreignobject_content * {
+    line-height:normal;
+}
+/* will have been sized by FO, so override the explicit size; AND shift back to right */
+.ltx_foreignobject_content .ltx_minipage {
+    width:110% !important;
+    position:relative; left:5%}
```

Comment:
Re font sizes: I believe within SVG we should prioritise matching font sizes, as in, passing an explicit `font-size:NNpt` as an inline style. It wouldn't be enough, so +10% is a good idea, but it doesn't make sense to support using completely different nominal sizes.